packages/starrating/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@pluralsight/ps-design-system-starrating",
+  "version": "1.0.0",
+  "description": "Design System component for star rating ui",
+  "main": "index.js",
+  "module": "index.js",
+  "scripts": {
+    "build": "run-s build:js build:css",
+    "build:css": "build-css",
+    "build:js": "babel src --out-dir dist --ignore spec.js --copy-files",
+    "build:watch": "npm run build:js -- --watch",
+    "prepublish": "npm run build",
+    "test": "jest",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "test:watch": "npm run test -- --watchAll",
+    "test:updateSnapshot": "npm run test -- --updateSnapshot"
+  },
+  "keywords": [],
+  "author": "travisbumgarner",
+  "license": "Apache-2.0",
+  "repository": "pluralsight/design-system",
+  "peerDependencies": {
+    "@pluralsight/ps-design-system-normalize": "^3.0.24",
+    "glamor": "^2.20.0",
+    "react": ">=0.15.0 < 17.0.0"
+  },
+  "dependencies": {
+    "@pluralsight/ps-design-system-core": "^4.3.3",
+    "@pluralsight/ps-design-system-theme": "^1.3.2",
+    "polished": "^2.3.0",
+    "prop-types": "^15.5.10"
+  },
+  "devDependencies": {
+    "@pluralsight/ps-design-system-build": "^1.8.3",
+    "@pluralsight/ps-design-system-storybook-addon-theme": "^1.1.44",
+    "@storybook/addon-storyshots": "3.2.14",
+    "@storybook/addons": "3.2.14",
+    "@storybook/react": "3.2.14",
+    "babel-cli": "^6.24.1",
+    "babel-preset-env": "^1.4.0",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "jest": "^21.2.1",
```

Comment:
Dane has been upgrading these versions.  He'll take care of this.

Reply:
@danethurber Does it make sense to just leave this like this or is there a version I should revert it to?

Reply:
I'm not sure I'll be able to update snapshots on my local machine if I revert this.

Reply:
all the other packages have been upgraded to jest@23. i've done quite a bit of work to the project and have sped up installs and bootstrapping so it should be faster. give me an hour or so and i'll send you a pr

Reply:
pr opened -> https://github.com/TravisBumgarner/design-system/pull/1

let's compare together :)